app/views/tutorials/show.html.erb

```diff
@@ -0,0 +1,180 @@
+<style>
```

Comment:
This entire tutorial page just outlines the information that was detailed and reviewed in the Readme.  Consistency here is good.